### PR TITLE
Desktop version of portfolio

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
# Desktop Version of Portfolio
## New Features:
✅ Add desktop version
✅ Add menu bar functionality.
✅ Create two new CSS files: _general.css_ and _responsive.css_ inside the CSS directory.
## Changes Made:
✅ Apply all the requirements that are asked.
✅ Use `flex-box` and `grid `layout in an a wide spread.

### To the code reviewer:
- [ ] I kindly ask dear code reviewer that if there is any kind of issue in this project, please do list them in a descriptive manner and give your best suggestions if needed.
- [ ] If you think there is some big issues that is essential to be changed please kindly contact me through **Zoom** or **Slack**, I'm available on Microverse UTC-6 time zone.

### CODE REVIEW

File: css/responsive.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+@media (min-width: 1020px) {
+  .section {
+    padding-right: 10rem;
+    padding-left: 10rem;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile-only {
+    display: none;
```
Comment (GedeonTS) [inline css/responsive.css:10]: - Good work implementing the desktop version of your Portfolio. However, according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to replicate the Figma template as much as possible. Please change the font sizes for your navbar and greeting text of the desktop version to match the Figma design

**Figma Design**
Screenshot from 2023-05-04 20-12-01

**Your Design**
Screenshot from 2023-05-04 20-02-45

File: index.html [hunk 2]
```diff
@@ -79,16 +81,29 @@ <h1 class="heading-primary">
...
+    <!-- MAIN PART OF THE PAGE -->
+    <!--------------------------->
+    <main class="bg-color main">
       <!----------------------->
       <!-- WORKS SECTION -->
       <!----------------------->
-      <section class="section-works">
+      <section class="section-works section section--lg">
         <div class="works-header">
           <h2 class="heading-secondary">My Recent <br />Works</h2>
```
Comment (GedeonTS) [inline index.html:101]: - According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to match the Figma design for the layout of this title on the desktop version

**Your Design**
Screenshot from 2023-05-04 20-03-34

**Figma Design**
Screenshot from 2023-05-04 20-12-15

File: index.html [hunk 3]
```diff
@@ -285,22 +297,43 @@ <h3 class="heading-tertiary">Skills</h3>
...
+        <img
+          src="imgs/desktop-version/illustrations/about-top-right.png"
+          alt="Beautiful illustration"
+          class="desktop-only position-absolute position--top position--right"
+        />
       </section>
 
       <!----------------------->
       <!-- CONTACT ME SECTION -->
       <!----------------------->
```
Comment (GedeonTS) [inline index.html:316]: On the contact Section, please make sure to follow the layout of elements from the Figma design according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.):

1. According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to respect the margins between the form and the footer.
2. Kindly make sure to add the line before the Footer

**Your Design**
Screenshot from 2023-05-04 20-07-41

**Figma Design**
Screenshot from 2023-05-04 20-07-24

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @M-Anwar-Hussaini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good PR Description :+1: 
- Linters are passing :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by M-Anwar-Hussaini** [comment]
# New Changes Applied:
☑️ Replicate the Figma desktop version templates.
- Change nav bar items `font-size=16px`;
- Change greeting text properties: `font-size=48px;` and `font-weight=800`;

☑️ Fix the **My Recent Work** text.
☑️ Add margin to the top of the footer.
☑️ Add a light narrow bar to the top of the footer section.
☑️ Apply the project to look as the same as the Figma template.

## To New Code Reviewer:
If there is still something that should be changed on this project, Kindly let me know them.
If you have time you can contact me as well, I would be very grateful of you.
### Contact Me:
- ✔️ **GitHub:** [@M-Anwar-Hussaini](https://github.com/M-Anwar-Hussaini/)
- ✔️ **Slack:** Anwar-Hussaini 
- ✔️ **Email:** [Anwar Hussaini](anwarhussaini160@gmail.com)
- ✔️ **LinkedIn:** [Mohammad Anwar Hussaini](https://www.linkedin.com/in/mohammad-anwar-hussaini-876638267/)
- ✔️ **WhatsApp:** +93 76 459 7266

**Review by braincee — APPROVED** [review]
Hi @M-Anwar-Hussaini , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**